src/ripple/rpc/impl/Utilities.cpp

```diff
@@ -45,24 +42,18 @@ addPaymentDeliveredAmount (
     if (transactionMeta && transactionMeta->hasDeliveredAmount ())
     {
         meta[jss::delivered_amount] =
-                transactionMeta->getDeliveredAmount ().getJson (1);
+            transactionMeta->getDeliveredAmount ().getJson (1);
         return;
     }
 
-    if (auto ledger = context.netOps.getLedgerBySeq (transaction->getLedger ()))
+    // Ledger 4594095 is the first ledger in which the DeliveredAmount field
```

Comment:
It'd be nice if we had a document identifying all these "magic times" - we have at least one more in the system right?